pkg/client/k8s.go

```diff
@@ -743,3 +743,21 @@ func (k K8sClient) GetPods(labelSelector string, fieldSelector string, namespace
 	}
 	return pods, nil
 }
+
+/*
+	LOCAL VOLUMES SPECIFIC METHODS
+*/
+
+// GetLocalPvDeployment returns the deployments with a specific
+// openebs-component-name label key
+func (k K8sClient) GetLocalPvDeployment(name string) (*appsv1.Deployment, error) {
```

Comment:
This, function can be made generic by, taking the labelSelector as input. Please make sure the code you write can be reused later.

Reply:
Yeah, sounds good. There are some other functions also that can be optimized in the same way.